src/main/java/gov/nih/nci/evs/api/fhir/R4/ConceptMapProviderR4.java

```diff
@@ -126,7 +127,16 @@ public Parameters translateInstance(
     try {
       FhirUtilityR4.mutuallyRequired("code", code, "system", system);
       FhirUtilityR4.mutuallyExclusive("target", target, "targetSystem", targetSystem);
-
+      for (final String param :
+          new String[] {"conceptMap", "coding", "codableConcept", "dependency"}) {
+        FhirUtilityR4.notSupported(request, param);
+      }
```

Comment:
See other comment, "codeableConcept" and "conceptMap" likely don't need to be here.